### PR TITLE
feat: fastly.sdkVersion implementation

### PR DESCRIPTION
**Pull request by guybedford**
This implements a new `fastly.sdkVersion` property as well as an equivalent `import { sdkVersion } from 'fastly:experimental'` version property.

This is immediately useful in the StarlingMonkey builds to be able to easily inspect if we are in a StarlingMonkey build by checking `fastly.sdkVersion === 'starlingmonkey-dev'`, which allows smoothing over any test handling edge cases more easily.

It's included in the types as a hidden property that is experimental. The engine build for the main runtime is set to the next patch version with a `-dev` suffix.

We will need to ensure we update this definition for each version release.

Resolves https://github.com/fastly/js-compute-runtime/issues/775.

### CODE REVIEW

**Comment by guybedford** [comment]
@elliottt as discussed today, I've renamed this to `sdkVersion`. Ready for review.

File: runtime/js-compute-runtime/js-compute-builtins.h [hunk 1]
```diff
@@ -32,6 +32,8 @@ const JSErrorFormatString js_ErrorFormatString[JSErrNum_Limit] = {
 
 #include "host_interface/host_api.h"
 
+#define RUNTIME_VERSION "3.13.2-dev"
```
Comment (elliottt) [inline runtime/js-compute-runtime/js-compute-builtins.h:35]: We talked about this briefly earlier, but it would be great to figure out how to get `release-please` to keep this in sync.

**Review by elliottt — APPROVED** [review]
Looks great to me!

File: runtime/js-compute-runtime/js-compute-builtins.h (hunk 1 above)
Comment (guybedford) [reply runtime/js-compute-runtime/js-compute-builtins.h:35]: Will follow up as soon as I've worked out the release process.
